### PR TITLE
Add Sonarqube ruby script and appropriate files

### PR DESCRIPTION
**Pull request by lindgrenj6**
Pretty self explanatory. Similar to RedHatInsights/catalog-api#615

Link to results: https://sonarqube.corp.redhat.com/dashboard?id=InsightsPlatform%3Ajlindgren-approval-process

@bzwei If you would like me to move the sonar-scanner related files to another folder just let me know, not sure what you would prefer from an organization standpoint.

### CODE REVIEW

**Comment by lindgrenj6** [comment]
@miq-bot add_reviewer bzwei

**Comment by lindgrenj6** [comment]
Marking as WIP for now until the catalog-api pr gets merged - we'll do all the changes there and then percolate them throughout the others.

**Comment by bzwei** [comment]
The location seems file to me

**Comment by lindgrenj6** [comment]
@bzwei this is ready to be reviewed.

@gmcculloug

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/.scannerwork
+/target/
```
Comment (gmcculloug) [inline .gitignore]: @lindgrenj6 What ends up in this `target` directory?  Should it be more descriptive name?

Comment (lindgrenj6) [reply .gitignore]: Oh, that's just where maven puts its build files. Makes sense to ignore it, the `.gitignore` wasn't there before so I added it. I can remove it and add it in another PR if you'd like.

Comment (gmcculloug) [reply .gitignore]: Ok, not a problem.  You can leave it.

**Pull request merged**